repos/fdbt-site/src/pages/viewTimeRestrictions.tsx

```diff
@@ -0,0 +1,151 @@
+import { getAndValidateNoc, sentenceCaseString } from '../utils';
+import { getTimeRestrictionByNocCode } from '../data/auroradb';
+import SubNavigation from '../layout/SubNavigation';
+
+const title = 'Time restrictions';
+const description = 'View and edit your time restrictions.';
+
+const dayMappings = {
+    monday: 'Mon',
+    tuesday: 'Tue',
+    wednesday: 'Wed',
+    thursday: 'Thu',
+    friday: 'Fri',
+    saturday: 'Sat',
+    sunday: 'Sun',
+    bankHoliday: 'BH',
+};
+
+interface TimeRestrictionProps {
+    timeRestrictions: PremadeTimeRestriction[];
+}
+
+const formatTime = (time: string): string => (time ? `${time.substring(0, 2)}:${time.substring(2, 4)}` : '');
+
+const formatTimeBand = (timeBand: TimeBand): string =>
+    `${formatTime(timeBand.startTime)}–${formatTime(timeBand.endTime)}`;
+
+const formatTimeBands = (timeBands: TimeBand[]): string =>
+    timeBands.length > 0
+        ? timeBands
+              .map((timeBand, index) => (index === 0 ? formatTimeBand(timeBand) : `, ${formatTimeBand(timeBand)}`))
+              .join('')
+        : 'Valid all day';
+
+const formatDayRestriction = (timeRestriction: PremadeTimeRestriction, day: string): JSX.Element => {
+    const matchedDayRestriction = timeRestriction.contents.find((dayRestriction) => dayRestriction.day === day);
+    return matchedDayRestriction ? (
+        <span className="day-restriction">{formatTimeBands(matchedDayRestriction.timeBands)}</span>
+    ) : (
+        <span className="day-restriction not-valid">Not valid</span>
+    );
+};
+
+const ViewTimeRestrictions = ({ timeRestrictions }: TimeRestrictionProps): ReactElement => (
+    <BaseLayout title={title} description={description} showNavigation={true}>
+        <div className="govuk-width-container">
+            <main className="govuk-main-wrapper">
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-one-third">
+                        <SubNavigation />
+                    </div>
+
+                    <div className="govuk-grid-column-two-thirds">
+                        <h1 className="govuk-heading-xl">Time restrictions</h1>
+                        <p className="govuk-body">
+                            Define certain days and time periods that your tickets can be used within
+                        </p>
+
+                        {!timeRestrictions.length ? (
+                            <NoTimeRestrictions />
+                        ) : (
+                            <TimeRestrictions timeRestrictions={timeRestrictions} />
+                        )}
+                    </div>
+                </div>
+            </main>
+        </div>
+    </BaseLayout>
+);
+
+const NoTimeRestrictions = (): ReactElement => {
+    return (
+        <div className="govuk-heading-m">
+            <p className="govuk-body">
+                <em>You currently have no time restrictions saved.</em>
```

Comment:
On the passenger type page we currently don't have this text in italics, I think it might be best to not do it here either just to keep it consistent. What do you think @matt-twomey ?

Reply:
Updated passenger type to use italics as specified in Jira ticket